src/tests/ftest/util/soak_test_base.py

```diff
@@ -569,10 +593,22 @@ def run_soak(self, test_param):
 
         # cleanup soak log directories before test
         try:
-            run_local(self.log, f"rm -rf {self.sharedsoak_dir}/*", timeout=30)
+            run_local(self.log, f"rm -rf {self.soak_dir}/*", timeout=300)
         except RunException as error:
             raise SoakTestError(
-                f"<<FAILED: Soak directory {self.sharedsoak_dir} was not removed>>") from error
+                f"<<FAILED: Log directory {self.soak_dir} was not removed>>") from error
+        if self.enable_remote_logging:
+            result = run_remote(
+                self.log, self.hostlist_clients, f"rm -rf {self.soak_dir}/*", timeout=300)
+            if not result.passed:
+                raise SoakTestError(
+                    f"<<FAILED: Log directory not removed from clients>> {result.log_output}")
```

Comment:
Sorry I missed this earlier - same issue as before:
```suggestion
                    f"<<FAILED: Log directory not removed from {str(result.failed_hosts)}>>")
```

Reply:
hmm missed it too...

Reply:
@phender Looks like this is resolved now